corpus/macros.txt

```diff
@@ -31,56 +47,81 @@ a!('lifetime)
 default!(a);
 union!(a);
 
-----
+--------------------------------------------------------------------------------
 
 (source_file
   (macro_invocation
     (identifier)
-    (token_tree (identifier)))
+    (token_tree
+      (non_special_punctuation)
+      (identifier)
+      (non_special_punctuation)))
   (macro_invocation
     (identifier)
-    (token_tree (identifier)))
+    (token_tree
+      (non_special_punctuation)
+      (identifier)
+      (non_special_punctuation)))
   (macro_invocation
     (identifier)
-    (token_tree (identifier)))
+    (token_tree
+      (non_special_punctuation)
+      (identifier)
+      (non_special_punctuation)))
   (macro_invocation
     (identifier)
-    (token_tree (identifier) (identifier)))
+    (token_tree
+      (identifier)
+      (non_special_punctuation)
+      (identifier)
+      (non_special_punctuation)
+      (non_special_punctuation)))
   (macro_invocation
     (identifier)
-    (token_tree (char_literal) (char_literal)))
+    (token_tree
+      (char_literal)
+      (non_special_punctuation)
+      (char_literal)))
   (macro_invocation
     (identifier)
-    (token_tree (char_literal) (char_literal)))
+    (token_tree
+      (char_literal)
+      (non_special_punctuation)
+      (char_literal)))
   (macro_invocation
     (identifier)
-    (token_tree (identifier)))
+    (token_tree
+      (identifier)))
   (macro_invocation
     (identifier)
-    (token_tree (identifier)))
+    (token_tree
+      (identifier)))
   (macro_invocation
     (identifier)
-    (token_tree (identifier))))
+    (token_tree
+      (identifier))))
 
-=====================================
+================================================================================
 Macro invocation with comments
-=====================================
+================================================================================
 
 ok! {
   // one
   /* two */
 }
 
----
+--------------------------------------------------------------------------------
 
 (source_file
   (macro_invocation
     (identifier)
-    (token_tree (line_comment) (block_comment))))
+    (token_tree
```

Comment:
Again, could you remove all those indentation changes that makes it more difficult to review this PR.
Remove all unrelated changes.